Sources/NostrSDK/Events/Calendars/CalendarNostrEvent.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  CalendarNostrEvent.swift
+//
+//
+//  Created by Terry Yiu on 12/1/23.
+//
+
+import Foundation
+
+/// A calendar is a collection of calendar events, represented as a custom replaceable list event.
+/// A user can have multiple calendars.
+/// One may create a calendar to segment calendar events for specific purposes. e.g., personal, work, travel, meetups, and conferences.
+///
+/// See [NIP-52 - Calendar](https://github.com/nostr-protocol/nips/blob/master/52.md#calendar).
+public final class CalendarNostrEvent: NostrEvent, IdentifierTagInterpreting, TitleTagInterpreting {
```

Comment:
Just curious: why did you choose `CalendarNostrEvent` rather than `CalendarEvent` or maybe `CalendarListEvent`?

Reply:
I'm trying to avoid overloading the term "event", which we use to describe Nostr generically, but also "calendar events". I really just want to call this Calendar but I wanted to avoid naming collisions with Calendar from the standard Swift API. Maybe CalendarListEvent is better.